### PR TITLE
League female representation

### PR DESCRIPTION
**Pull request by mohit-codecombat**
Added rotational female representation with transition.

Attaching video

https://user-images.githubusercontent.com/75598717/111490716-425e8400-8761-11eb-9fc9-f5260326572d.mp4

https://user-images.githubusercontent.com/75598717/111490722-44c0de00-8761-11eb-80d0-fe24a21c929f.mp4

### CODE REVIEW

**Comment by nwinter** [comment]
As discussed, tried another approach for the fading; what do we think of this?

Also, I couldn't get the `beforeUnmount` to work, maybe n00b Vue mistake.

**Comment by mohit-codecombat** [comment]
I am able call `clearInterval()` method from `beforeDestroy` and it is calling when we go to other page from `/league` page.

File: app/views/landing-pages/league/PageLeagueGlobal.vue [hunk 1]
```diff
@@ -278,7 +293,7 @@ export default {
       if (image) {
         return `/file/${image}`
       }
-      return `/images/pages/league/graphic_1.png`
+      return `/images/pages/league/student_hugging.png`
```
Comment (nwinter) [inline app/views/landing-pages/league/PageLeagueGlobal.vue:296]: We should either rename the images to have the same names as before, or we should remove the old unused images (I think game_hero.png, graphic_1.png, and graphic_hugging.png).

**Review by nwinter — CHANGES_REQUESTED** [review]
Looking pretty good. The new layout on mobile doesn't work as well on a team-page like /league/team-derbezt with a custom image:
Screen Shot 2021-03-18 at 05 33 18

Should create more space between the team image and this. We could also ditch the AI League logo element on mobile if we want, and/or shrink this rotating hero element or the header; together they're making it take a lot of scrolling before getting to the meat (the team-specific details).

File: app/views/landing-pages/league/PageLeagueGlobal.vue (hunk 1 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/league/PageLeagueGlobal.vue:296]: Removed unused images `game_hero.png`, `graphic_1.png`, and `graphic_hugging.png`

**Comment by mohit-codecombat** [comment]
Attaching screenshot after updating for `/league/team-derbezt`

Competitive AI c (5)

File: app/views/landing-pages/league/PageLeagueGlobal.vue [hunk 2]
```diff
@@ -898,6 +920,25 @@ export default {
...
     max-width: 510px;
   }
+  .hero-rotation {
+    display: flex;
+    justify-content: flex-end;
+    position:absolute;
+    top: 0px;
+    width: 65%;
+    right: 0px;
+    img{
```
Comment (codeluggage) [inline app/views/landing-pages/league/PageLeagueGlobal.vue:930]:
```suggestion
    img {
```

**Review by codeluggage — APPROVED** [review]
Looks great from what I can see, and the context I have 👍  👍  

Love that we're getting `style`'s out of the HTML and into the CSS as we're updating things 👌

**Pull request merged**